calibration/CTDMOG/CGINS-CTDMOG-13428__20161025.csv

```diff
@@ -22,4 +22,4 @@ serial,name,value,notes
 37-13428,CC_ptempa0,1.883382e+002,
 37-13428,CC_ptempa1,-7.719570e-002,
 37-13428,CC_ptempa2,1.080045e-006,
-37-13428,CC_wbotc,9.2946e-008,
+37-13428,CC_wbotc,9.294638e-08,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT